### ISSUE TITLE
[DOC request] Can we include some comparisons with existing ESB's in the wild? EOM

### ISSUE THREAD

**Issue by QuantumInformation** [salboaie/SwarmESB#16]

**Comment by salboaie** [comment]
Yes. We will start in July a big research project (5 years project) to grow SwarmESB as a integration platform and such comparations would be very good. I will allocate resources in this direction. Do you have some plans,idea about this?

**Comment by QuantumInformation** [comment]
I've used Mule ESB for 6 months, its pretty much dominant due to the amount of connectors it has. Swarm would need to prove its worth in a busy market.
